node/src/sub_lib/utils.rs

```diff
@@ -119,19 +119,21 @@ fn crash_request_analyzer(
     crashable: bool,
     crash_key: &str,
 ) -> Option<UiCrashRequest> {
-    match (crashable, UiCrashRequest::fmb(msg.body)) {
-        (false, _) => None,
-        (true, Err(_)) => None,
-        (true, Ok((msg, _))) if msg.actor == crash_key => Some(msg),
-        (true, Ok((msg, _))) => {
-            debug!(
-                logger,
-                "Rejected crash instruction for '{}' with message '{}'",
-                msg.actor,
-                msg.panic_message
-            );
-            None
+    if !crashable {
+        if logger.debug_enabled() {
```

Comment:
I guess I'll leave it unchanged. The reason is at the additional condition that we want to print a log only if the actor is the right one otherwise we don't care but even in such a case we would have to give out a String because logger.debug() requires it. And we wouldn't have where to find any good solution...